README.md

```diff
@@ -113,3 +135,40 @@ Client Implementation
 
 `pint`, our Python-based client, is available in the Public Cloud Module, or in
 the Cloud:Tools project in OBS, as `python-susepubliccloudinfo`.
+
+Development
+-----------
+The following steps are only recommended if you want to build this project from sources, work on the codebase or test the latest development changes
+
+  1. **Install Git**
+
+         $ sudo zypper in git
+
+  2. **Install basic Ruby environment**
+
+         $ sudo zypper in ruby rubygem-bundler
+
+  3. **Install dependencies**
+
+     Install packages needed to compile Gems with native extensions:
+
+         $ sudo zypper in gcc make ruby-devel ruby2.5-rubygem-rack
+
+  4. **Clone public-cloud-info-service repository and install Gem dependencies**
+
+         $ git clone https://github.com/SUSE-Enceladus/public-cloud-info-service.git 
+         $ cd public-cloud-info-service 
+         $ bundle install
+
+  6. **In order to run REST APIs on the host**
+        
+         $ cd public-cloud-info-service
+         $ export FRAMEWORKS=spec/fixtures/framework*.xml
+         $ rackup
+         
+      To run REST APIs, point your browser to https://127.0.0.1:9292/
```

Comment:
https gets me ssl errors consistently, guessing this isnt setup with a certificate for the local environment... works fine with plain http

Reply:
No, sinatra does not inherently process TLS; this has to be handled though an additonal module or (most commonly) via a reverse proxy.